pyspecdata/core.py

```diff
@@ -7508,6 +7508,7 @@ def _init_datanode(self, datanode, **kwargs):
         if "axes" in list(datadict.keys()):
             myaxiscoords = [None] * len(mydimlabels)
             myaxiscoordserror = [None] * len(mydimlabels)
+            myaxis_units = [None] * len(mydimlabels)
```

Comment:
I think this handles what you are talking about [here](https://github.com/jmfrancklab/pyspecdata/pull/50/files#r1688471252)